ibtk/include/ibtk/LaplaceOperator.h

```diff
@@ -78,6 +78,11 @@ class LaplaceOperator : public LinearOperator
     virtual void setPoissonSpecifications(const SAMRAI::solv::PoissonSpecifications& poisson_spec);
 
     /*!
+     * \brief Set the scale factor for C and D in the SAMRAI::solv::PoissonSpecifications object.
+     */
+    virtual void setCoefScales(double C_scale, double D_scale);
```

Comment:
I guess this is to make it easy to rescale variable coefficient cases. It seems potentially confusing for the case of constant coefficients, however. Worth thinking about whether there is a better way to accomplish this.